internal/controller/eventing/subscription/jetstream/matchers_test.go

```diff
@@ -35,16 +35,16 @@ func BeJetStreamSubscriptionWithSubject(source, subject string,
 		if err != nil {
 			return false, err
 		}
-		js := jetstream.JetStream{
+		jetstream := jetstream.JetStream{
```

Comment:
This shadows the package name `jetstream`.

Reply:
fixed